src/main/java/com/iexec/sms/admin/AdminService.java

```diff
@@ -36,13 +41,16 @@ public class AdminService {
     private final String datasourceUrl;
     private final String datasourceUsername;
     private final String datasourcePassword;
+    private final String storageFolder;
 
     public AdminService(@Value("${spring.datasource.url}") String datasourceUrl,
                         @Value("${spring.datasource.username}") String datasourceUsername,
-                        @Value("${spring.datasource.password}") String datasourcePassword) {
+                        @Value("${spring.datasource.password}") String datasourcePassword,
+                        @Value("${spring.datasource.storage-folder}") String storageFolder) {
```

Comment:
I don't think this is the right directory

Reply:
Please see 5a68218